test/lit.cfg.py

```diff
@@ -74,7 +74,6 @@
     "PYTHONPATH",
     [
         os.path.join(config.mpact_obj_root, "python_packages/mpact"),
-        os.path.join(config.torch_mlir_obj_root, "python_packages/torch_mlir"),
```

Comment:
I find it strange that we can remove this now?

It looks like you moved all the torch-mlir packages into the mpact build? Can't we keep the two separate?